### PR TITLE
Add support for CORS

### PR DESCRIPTION
**Pull request by tokland**
Closes EyeSeeTea/FIRE-WiFiCalling#15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,21 @@ You can use `curl` with basic authentication (`-u user:password`, see `models.us
...
 ```
 
+Example from AngularJS:
+
+```
+const url = "http://dev.eyeseetea.com:5000/users/3";
+const headers = {'Authorization': 'Basic ' + btoa("joel:joel1234")};
+http.get(url, {headers}).subscribe(response => console.log(response.json()));
+```
+
```
Comment (adrianq) [inline README.md:37]: @tokland Many Many Thanks! :)

**Review by adrianq — APPROVED** [review]
Thanks @tokland! Looks great! Have you already deployed it on dev-est? I am merging it so this code gets to master

File: README.md (hunk 1 above)
Comment (tokland) [reply README.md:37]: Better 1 person to break its head to make it work than N :)

**Comment by tokland** [comment]
Sorry, I  forgot to mention it. Yes, deployed to dev.est, examples already working.

File: README.md (hunk 1 above)
Comment (tokland) [reply README.md:37]: btw, observables returned by `http.get` are very basic, to get more operators than `subscribe`: `import 'rxjs/Rx';`

**Pull request merged**